### PR TITLE
Swift 2 and Xcode 7 fixes

### PR DESCRIPTION
**Pull request by steveriggins**

### CODE REVIEW

File: THGHybridWeb.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -76,11 +151,11 @@
...
 		17E076E21B024AC4008F89EC /* UIWebView+JavaScriptContext.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = "UIWebView+JavaScriptContext.swift"; sourceTree = "<group>"; };
 		17E076E91B025AA2008F89EC /* WebViewControllerTests.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = WebViewControllerTests.swift; sourceTree = "<group>"; };
+		205A0E3E1B4DEAB400D72DCF /* THGBridge.xcodeproj */ = {isa = PBXFileReference; lastKnownFileType = "wrapper.pb-project"; name = THGBridge.xcodeproj; path = ../BridgeOfDeath/THGBridge.xcodeproj; sourceTree = "<group>"; };
+		205A0E4F1B4DEAE000D72DCF /* THGFoundation.xcodeproj */ = {isa = PBXFileReference; lastKnownFileType = "wrapper.pb-project"; name = THGFoundation.xcodeproj; path = ../Excalibur/THGFoundation.xcodeproj; sourceTree = "<group>"; };
+		205A0E5B1B4DEAEE00D72DCF /* THGLog.xcodeproj */ = {isa = PBXFileReference; lastKnownFileType = "wrapper.pb-project"; name = THGLog.xcodeproj; path = ../Shubbery/THGLog.xcodeproj; sourceTree = "<group>"; };
```
Comment (angelodipaolo) [inline THGHybridWeb.xcodeproj/project.pbxproj]: Shubbery?

Comment (steveriggins) [inline THGHybridWeb.xcodeproj/project.pbxproj]: haha oops, looks like this has been wrong for awhile

Comment (steveriggins) [inline THGHybridWeb.xcodeproj/project.pbxproj]: Hmm it looks right on the main branch of walmart

Comment (steveriggins) [inline THGHybridWeb.xcodeproj/project.pbxproj]: go ahead and approve all of these and I will push up a global fix to all tomorrow. Looks like cody fixed it on a newer walmart branch already

**Pull request merged**